client/js/components/content-slider.js

```diff
@@ -365,6 +364,14 @@ const contentSlider = (el, options) => {
   sliderElements.prevControl.addEventListener('click', prevSlide, true);
   sliderElements.nextControl.addEventListener('click', nextSlide, true);
 
+  nodeList(sliderElements.slideItems).forEach((item) => {
+    item.addEventListener('click', ({ target, currentTarget }) => {
+      if (!target.matches('img')) return;
```

Comment:
The captions are part of the elements as well (and we require clicking the +more button to perform a separate function).